### ISSUE TITLE
type.any() returning error at createModel()

### ISSUE THREAD

**Issue by bplatz** [neumino/thinky#209]
I'm using 1.16.10 and getting errors registering a schema with type.any() in it. Here is my schema that is throwing the error:
```
import thinky from '../lib/thinky';
var type = thinky.type;

var Store = thinky.createModel('Store', {
    id: type.string(),
    value: type.any(),
    date: type.date(),
    vendor: type.string()
});
export default Store;
```
And here is the relevant part of the stack trace:
```
Document failed validation: The value must be `String`/`Number`/`Boolean`/`Date`/`Buffer`/`Object`/`Array`/`'virtual'`/`'Point'` for [value][_default]
    at parse (~/node_modules/thinky/lib/schema.js:276:11)
    at ~/node_modules/thinky/lib/schema.js:228:28
    at Object.loopKeys (~/node_modules/thinky/lib/util.js:203:16)
    at parse (~/node_modules/thinky/lib/schema.js:227:12)
    at ~/node_modules/thinky/lib/schema.js:228:28
    at Object.loopKeys (~/node_modules/thinky/lib/util.js:203:16)
    at Object.parse (~/node_modules/thinky/lib/schema.js:227:12)
    at new Model (~/node_modules/thinky/lib/model.js:34:29)
    at Function.Model.new (~/node_modules/thinky/lib/model.js:95:15)
    at Thinky.createModel (~/node_modules/thinky/lib/thinky.js:126:24)
    at Object.<anonymous> (~/source/models/store.js:9:20)
```

**Comment by neumino** [comment]
Fix in 1.16.11 -- not sure what happened here but I suspect a bad merge. I'm sure I wrote a test for that and couldn't find it -_-

**Comment by bplatz** [comment]
Thanks for the quick fix!
